src/DataStructures/DataBoxTag.hpp

```diff
@@ -414,7 +423,7 @@ struct is_variables_compute_item<
 template <template <typename...> class Wrapper, typename TagList,
           typename... Args>
 using wrap_tags_in =
-    tmpl::transform<TagList, tmpl::bind<Wrapper, tmpl::_1, Args...>>;
+    tmpl::transform<TagList, tmpl::bind<Wrapper, tmpl::_1, tmpl::pin<Args>...>>;
```

Comment:
is this related to the commit message?

Reply:
Indirectly, yes. I'll split it into a separate commit